R/RocksDB/build_tarballs.jl

```diff
@@ -0,0 +1,74 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "RocksDB"
+version = v"8.9.1"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/facebook/rocksdb.git", "49ce8a1064dd1ad89117899839bf136365e49e79"),
+    ArchiveSource("https://github.com/phracker/MacOSX-SDKs/releases/download/10.15/MacOSX10.15.sdk.tar.xz",
+                  "2408d07df7f324d3beea818585a6d990ba99587c218a3969f924dfcc4de93b62"),
+    DirectorySource("./bundled"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd rocksdb
+
+# Apply patch for arm and mingw
+if [[ "${target}" = arm* ]] || [[ "${target}" == aarch* ]]; then
+    atomic_patch -p1 ../patches/arm64.patch
+fi
+
+mkdir build && cd build
+
+# Resolve: error: aligned allocation function of type 'void *(std::size_t, std::align_val_t)' is only available on macOS 10.13 or newer
+if [[ "${target}" == x86_64-apple-darwin* ]]; then
+    echo "Installing newer MacOS 10.15 SDK"
+    export MACOSX_DEPLOYMENT_TARGET=10.15
+    # ...and install a newer SDK which supports `std::filesystem`
+    pushd $WORKSPACE/srcdir/MacOSX10.*.sdk
+    rm -rf /opt/${target}/${target}/sys-root/System
+    cp -ra usr/* "/opt/${target}/${target}/sys-root/usr/."
+    cp -ra System "/opt/${target}/${target}/sys-root/."
+    popd
+fi
+
+cmake -DCMAKE_INSTALL_PREFIX=${prefix} \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DWITH_GFLAGS=0 \
+    -DROCKSDB_BUILD_SHARED=1 \
+    -DPORTABLE=1 \
+    -DWITH_TOOLS=0 \
+    -DWITH_TESTS=0 \
+    -DWITH_BENCHMARK_TOOLS=0 \
+    -DCMAKE_BUILD_TYPE=Release ..
+
+cmake --build . --parallel ${nproc}
+cmake --install .
```

Comment:
> Unable to find valid license file in "${prefix}/share/licenses/RocksDB"

The auditor is saying it can't find a license in the installed files, so please add a call to `install_license` at the end of the script to install the license provided with the library.